src/app/cli/security_materials.cpp

```diff
@@ -0,0 +1,207 @@
+/*
+ *    Copyright (c) 2020, The OpenThread Commissioner Authors.
+ *    All rights reserved.
+ *
+ *    Redistribution and use in source and binary forms, with or without
+ *    modification, are permitted provided that the following conditions are met:
+ *    1. Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *    3. Neither the name of the copyright holder nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ *    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ *    AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ *    IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ *    ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ *    LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *    CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * @file
+ *   The file implements security materials storage.
+ */
+
+#include "app/cli/security_materials.hpp"
+#include <string>
+#include "app/cli/console.hpp"
+#include "common/error_macros.hpp"
+#include "common/file_util.hpp"
+#include "common/utils.hpp"
+
+namespace ot {
+
+namespace commissioner {
+
+namespace sm {
+
+using SMPair = std::pair<std::string, ByteArray *>;
+
+static Error GetNetworkSM_impl(const std::string  aNwkFolder, // a folder to start from
+                               const std::string  aAlias,     // network id or network name
+                               bool               aCCM,
+                               SecurityMaterials &aSM);
+
+static class SMRoot
+{
+    std::string mRootPath;
+
+public:
+    void        Set(const std::string aRoot) { mRootPath = aRoot; }
+    std::string Get() const { return mRootPath; }
+} smRoot;
+
+Error Init(const Config &aDefaultConfig)
+{
+    std::string root = aDefaultConfig.mThreadSMRoot;
+
+    if (root.empty())
+    {
+        root = STR_SAFE(getenv("THREAD_SM_ROOT"));
+        if (root.empty())
+        {
+            Console::Write("ThreadSMRoot value is not available", Console::Color::kYellow);
```

Comment:
I guess, the message may help investigate an issue with connectivity if any. So I prefer to have it just suppressed by being directed to log file instead of console. The decision is made upon **gVerbose** console flag state introduced in one of the further commits.